src/common/util_pmem.h

```diff
@@ -42,26 +42,33 @@ extern "C" {
 #endif
 
 #include "libpmem.h"
+#include "out.h"
 
 /*
  * util_persist -- flush to persistence
  */
-static inline void
+static inline int
 util_persist(int is_pmem, const void *addr, size_t len)
```

Comment:
we intentionally ignore the return value of msync, so I'd suggest simply casting the result of the msync to (void).